crates/sui-core/src/epoch/randomness.rs

```diff
@@ -241,29 +364,60 @@ impl RandomnessManager {
                 committee.epoch()
             );
             // Load intermediate data.
+
+            // load from the deprecated table if the binary was changed during dkg.
+            #[allow(deprecated)]
+            rm.processed_messages
```

Comment:
This should work just based on the protocol version and not always checking both tables, same as we do it for e.g. `pending_checkpoints_v2` table in `AuthorityPerEpochStore`

then once new protocol version is in use everywhere we can clean up the branch